webpack/scenes/ModuleStreams/Details/ModuleStreamDetailInfo.js

```diff
@@ -2,27 +2,35 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Table } from 'react-bootstrap';
 
-const displayMap = {
-  name: __('Name'),
-  stream: __('Stream'),
-  version: __('Version'),
-  arch: __('Arch'),
-  context: __('Context'),
-  description: __('Description'),
-  summary: __('Summary'),
-  uuid: __('UUID'),
+// using Map to preserve order
+const displayMap = new Map([
+  ['name', __('Name')],
+  ['summary', __('Summary')],
+  ['description', __('Description')],
+  ['stream', __('Stream')],
+  ['version', __('Version')],
+  ['arch', __('Arch')],
+  ['context', __('Context')],
+  ['uuid', __('UUID')],
+]);
+
+const createRows = (details, mapping) => {
+  const rows = [];
+  /* eslint-disable no-restricted-syntax, react/jsx-closing-tag-location */
+  for (const key of mapping.keys()) {
+    rows.push(<tr key={key}>
+      <td><b>{mapping.get(key)}</b></td>
+      <td>{details[key]}</td>
+    </tr>);
+  }
+  /* eslint-enable no-restricted-syntax, react/jsx-closing-tag-location */
```

Comment:
reasons for disabling these rules:

- no-restricted-syntax: using a for loop on a map object is [from MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Iterating_Maps_with_for..of). `.map()` doesn't work in this case 

- jsx-closing-tag-location: was fighting with this and another rule that didn't allow me to put the first `tr` tag on a new line